standards/version-control.md

```diff
@@ -0,0 +1,41 @@
+# Version Control
+These standards are based on Defra Digital's [version control standards](https://github.com/DEFRA/software-development-standards/blob/master/standards/version_control_standards.md), but are extended to cover not only the application version, but version control of Docker images and Helm charts.
+
```

Comment:
Might be worth saying somewhere that using the Jenkins pipeline library handles the Docker and Helm microservice versioning